naloge/dag.tex

```diff
@@ -33,6 +33,55 @@
 \podnaslov{Graf}
 \end{slika}
 \end{vprasanje}
+
 \begin{odgovor}
+\begin{enumerate}[(a)]
+\item Topološko ureditev dobimo z uporabo algoritma {\sc Topo}
+iz naloge~\res[topo]{}(a).
+Dobimo $[s, b, a, d, h, e, c, f, i, g, t]$. 
+Graf v tej ureditvi si lahko ogledamo na sliki~\fig[dag-resitev].
+
+\item Najkrajšo pot od vozlišča $s$ do vozlišča $t$
+lahko izračunamo z algoritmom {\sc NajkrajšaPot} iz naloge~\res[topo]{}(b).
+Ta nam vrne pot $s - a - d - h - t$ dolžine $12$.
```

Comment:
Tukaj se moja rešitev ni ujemala s tvojo. Prosil bi te, če lahko narediš še tabelo s potekom izvajanja algoritma (in s tem preveriš tudi pravilnost rešitve).